### PR TITLE
implemented edit details method

### PR DESCRIPTION
**Pull request by tshet-eg**

### CODE REVIEW

File: BankManagementSystem/Controller/CustomerController.cs [hunk 1]
```diff
@@ -11,12 +12,70 @@ public void CreateAccount()
...
+                {
+                    case 1:
+                        Console.WriteLine("Enter new name:");
+                        string replaceName=Console.ReadLine();
+                        customerModel.CustomerName = replaceName;
+                        break;
+                    case 2:
+                        Console.WriteLine("Enter updated date of birth:");
+                        string updateDOB = Console.ReadLine();
+                        customerModel.CustomerName = updateDOB;
```
Comment (Vijeth-eg) [inline BankManagementSystem/Controller/CustomerController.cs]: change the customerModel.customerName to customerModel.CustomerDateOfBirth

File: BankManagementSystem/Controller/CustomerController.cs [hunk 2]
```diff
@@ -11,12 +12,70 @@ public void CreateAccount()
 
 
         }
-        public void EditDetails()
+        public void EditDetails(long accountNumber)
         {
+            CustomerModel customerModel;
+            customerModel = (CustomerModel)Program.CustomerTable[accountNumber];
+                Console.WriteLine("Saved account details:");
```
Comment (Vijeth-eg) [inline BankManagementSystem/Controller/CustomerController.cs]: add print message as "Account Details"

File: BankManagementSystem/Controller/CustomerController.cs (hunk 1 above)
Comment (tshet-eg) [reply BankManagementSystem/Controller/CustomerController.cs]: will check that

**Review by Vijeth-eg — APPROVED** [review]
good to go

**Pull request merged**